hm_pyhelper/miner_param.py

```diff
@@ -191,6 +191,40 @@ def get_ethernet_addresses(diagnostics):
             raise MinerFailedToFetchEthernetAddress(str(e))
 
 
+def get_temperatures(diagnostics):
+    path_to_temp_file = "/sys/class/thermal/thermal_zone0/temp"
```

Comment:
Does this work on both Raspberry Pi and ROCK Pis?

Reply:
This is standard on linux depending on hardware support. They are both supported in RPi for at least RPi2. More I can say has I do not have ssh access to the Nebra.